src/map/status.cpp

```diff
@@ -4227,6 +4228,10 @@ int status_calc_pc_(struct map_session_data* sd, enum e_status_calc_opt opt)
 	if( (skill = pc_checkskill(sd,SU_SPRITEMABLE)) > 0 && !sd->sc.data[SC_SPRITEMABLE] )
 		sc_start(&sd->bl, &sd->bl, SC_SPRITEMABLE, 100, 1, -1);
 
+	if( previous_st ){
+		script_attach_state(previous_st);
```

Comment:
Causes error in latest commit.

Reply:
Why would it? I think you merged badly.

Reply:
My bad it was causing the other error. I'l post it in conversation. Sorry.